### ISSUE TITLE
Flatten `examples/` directory

### ISSUE THREAD

**Issue by pi0** [nuxt/framework#7246]
(talking together with @Atinux). Currently, `examples/` dir is too complex to discover examples. Flattening names as before could be much better.

**Comment by Atinux** [comment]
I would even create a `nuxt/examples` repository directly and start working on more "real-life" example and using the stable release.

This way the `nuxt/framework` will be lighter and the community could easily contribute to `nuxt/examples` in the long run. WDYT?

**Comment by pi0** [comment]
Agreed. We can also potentially deploy them all and periodically test them against edge channel. (assigning myself to transfer git history into new repo)
